lib/resource/image_paths.dart

```diff
@@ -53,6 +53,9 @@ class ImagePaths {
   static String get icPersonCheck => _getImagePath('ic_person_check.svg');
   static String get icTwakeImageLogoBeta =>
       _getImagePath('ic_twake_image_beta.svg');
+  static String get logoPng => _getAssetPath('logo.png');
```

Comment:
why not use svg?

Reply:
- Reuse old logo already in source code
- Lib not support svg